demo/csi/aws/ebs/vault-example-setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Vault AWS Secrets Engine
+
+vault secrets enable -path=aws aws
+vault write aws/config/root \
+    access_key=A...NA \
+    secret_key=123XYZ \
+    region=us-east-1
+
+# Permissions needed by the EBS CSI driver
+
+vault write aws/roles/ebs-csi credential_type=iam_user \
+   policy_arns="arn:aws:iam::481516234200:policy/EBS-CSI" \
+   permissions_boundary_arn="arn:aws:iam::481516234200:policy/VaultGrantedBoundary"
```

Comment:
Similar to my comments elsewhere, let's remove any specific IDs from the example. If you do something like the following and `set -u`, it forces the user to enter the values.

```
vault write aws/roles/ebs-csi credential_type=iam_user \
   policy_arns="arn:aws:iam::${AWS_ACCOUNT_ID}:policy/EBS-CSI" \
   permissions_boundary_arn="arn:aws:iam::${AWS_ACCOUNT_ID}:policy/VaultGrantedBoundary"
```